### DISCUSSION TITLE
How do I use ffmpeg-cli the same way as subtitle extract in lossless-cut

### DISCUSSION THREAD

**Discussion by Issykul** [mifi/lossless-cut#1439] [Q&A] [answered]
I want to use the ffmpeg-cli on my server the same way as when I click "Extract this track as file" in the "Tracks" Menu in the top left corner when I do it on subtitles.

**Comment by mifi** [comment, marked as answer]
losslesscut is open source so you can see exactly how it works in the code :) just search for that text and try to trace your way thru the code

**Reply by Issykul** [reply]
Just now i have tried to look for a solution inside the repo, but i can't find the right part in the code. I am no Dev myself and definitely have no experience with javascript. That's why i asked here, because i was hoping that someone that know what to do can help me with the right handbrake-cli command.

**Reply by mifi** [reply]
ok i'll help you out:
```bash
ffmpeg -i input.mp4 -map 0:s -c copy subtitle.mks
```

**Reply by Issykul** [reply]
Thank you 💯 sent you a few bux via PP
